vars/Fedora-27.yml

```diff
@@ -0,0 +1,11 @@
+---
+__postgresql_version: "9.6"
+__postgresql_data_dir: "/var/lib/pgsql/data"
+__postgresql_bin_path: "/usr/bin"
+__postgresql_config_path: "/var/lib/pgsql/data"
+__postgresql_daemon: postgresql
+__postgresql_packages:
+  - postgresql
```

Comment:
This file can be removed since 27 is EOL

Reply:
Done.